browser/ui/webui/settings/brave_settings_localized_strings_provider.cc

```diff
@@ -79,616 +79,644 @@ const char16_t kEnableNftDiscoveryLearnMoreUrl[] =
-     IDS_SETTINGS_SITE_SETTINGS_LOCALHOST_ACCESS},
-    {"siteSettingsCategoryLocalhostAccess",
-     IDS_SETTINGS_SITE_SETTINGS_LOCALHOST_ACCESS},
-    {"siteSettingsLocalhostAccessAsk",
-     IDS_SETTINGS_SITE_SETTINGS_LOCALHOST_ACCESS_ASK},
-    {"siteSettingsLocalhostAccessBlock",
-     IDS_SETTINGS_SITE_SETTINGS_LOCALHOST_ACCESS_BLOCK},
-    {"siteSettingsLocalhostAccessBlockExceptions",
-     IDS_SETTINGS_SITE_SETTINGS_LOCALHOST_ACCESS_BLOCK_EXCEPTIONS},
-    {"siteSettingsLocalhostAccessAllowExceptions",
-     IDS_SETTINGS_SITE_SETTINGS_LOCALHOST_ACCESS_ALLOW_EXCEPTIONS},
-    {"braveGetStartedTitle", IDS_SETTINGS_BRAVE_GET_STARTED_TITLE},
-    {"siteSettingsShields", IDS_SETTINGS_SITE_SETTINGS_SHIELDS},
-    {"siteSettingsShieldsStatus", IDS_SETTINGS_SITE_SETTINGS_SHIELDS_STATUS},
-    {"siteSettingsShieldsUp", IDS_SETTINGS_SITE_SETTINGS_SHIELDS_UP},
-    {"siteSettingsShieldsDown", IDS_SETTINGS_SITE_SETTINGS_SHIELDS_DOWN},
-    {"siteSettingsShieldsDescription",
-     IDS_SETTINGS_SITE_SETTINGS_SHIELDS_DESCRIPTION},
-    {"appearanceSettingsBraveTheme",
-     IDS_SETTINGS_APPEARANCE_SETTINGS_BRAVE_THEMES},
-    {"appearanceSettingsShowBookmarksButton",
-     IDS_SETTINGS_APPEARANCE_SETTINGS_SHOW_BOOKMARKS_BUTTON},
-    {"appearanceSettingsLocationBarIsWide",
-     IDS_SETTINGS_APPEARANCE_SETTINGS_LOCATION_BAR_IS_WIDE},
-    {"appearanceSettingsShowBraveNewsButtonLabel",
-     IDS_SETTINGS_SHOW_BRAVE_NEWS_BUTTON_LABEL},
-    {"appearanceSettingsBookmarBar", IDS_SETTINGS_SHOW_BOOKMARK_BAR},
-    {"appearanceSettingsBookmarBarAlways",
-     IDS_SETTINGS_ALWAYS_SHOW_BOOKMARK_BAR_ALWAYS},
-    {"appearanceSettingsBookmarBarNTP",
-     IDS_SETTINGS_ALWAYS_SHOW_BOOKMARK_BAR_ON_NTP},
-    {"appearanceSettingsBookmarBarNever", IDS_SETTINGS_NEVER_SHOW_BOOKMARK_BAR},
-    {"appearanceSettingsBookmarBarAlwaysDesc",
-     IDS_SETTINGS_ALWAYS_SHOW_BOOKMARK_BAR_ALWAYS_DESC},
-    {"appearanceSettingsBookmarBarNTPDesc",
-     IDS_SETTINGS_ALWAYS_SHOW_BOOKMARK_BAR_ON_NTP_DESC},
-    {"appearanceSettingsBookmarBarNeverDesc",
-     IDS_SETTINGS_NEVER_SHOW_BOOKMARK_BAR_DESC},
-    {"appearanceSettingsShowAutocompleteInAddressBar",
-     IDS_SETTINGS_APPEARANCE_SETTINGS_SHOW_AUTOCOMPLETE_IN_ADDRESS_BAR},
-    {"appearanceSettingsUseTopSiteSuggestions",
-     IDS_SETTINGS_APPEARANCE_SETTINGS_USE_AUTOCOMPLETE_TOP_SITES},
-    {"appearanceSettingsUseHistorySuggestions",
-     IDS_SETTINGS_APPEARANCE_SETTINGS_USE_AUTOCOMPLETE_HISTORY},
-    {"appearanceSettingsUseBookmarkSuggestions",
-     IDS_SETTINGS_APPEARANCE_SETTINGS_USE_AUTOCOMPLETE_BOOKMARKS},
-    {"appearanceSettingsUseLeoSuggestions",
-     IDS_SETTINGS_APPEARANCE_SETTINGS_USE_AUTOCOMPLETE_LEO},
-    {"appearanceSettingsGetMoreThemes",
-     IDS_SETTINGS_APPEARANCE_SETTINGS_GET_MORE_THEMES},
-    {"appearanceBraveDefaultImagesOptionLabel",
-     IDS_SETTINGS_APPEARANCE_SETTINGS_BRAVE_DEFAULT_IMAGES_OPTION_LABEL},
+      {"importExtensions", IDS_SETTINGS_IMPORT_EXTENSIONS_CHECKBOX},
```

Comment:
This formatting must have been not done for cr102 @mkarolin? `npm run format` is producing this - happy to keep in this PR as I hope to merge today